PROShine/Views/BattleView.xaml.cs

```diff
@@ -0,0 +1,410 @@
+﻿using PROBot;
+using PROProtocol;
+using System;
+using System.Collections.Generic;
+using System.Windows;
+using System.Windows.Controls;
+using FontAwesome.WPF;
```

Comment:
The `using`s are not sorted alphabetically!!1 :cry: 
(This is just nitpicking, I know.)